app/questionPage.html

```diff
@@ -41,6 +43,43 @@
 <div id="question_page">
     <div class="container">
         <div class="row">
+            <button type="button" class="btn btn-info left instructions-button" data-toggle="modal" data-target="#instructionsModal">Instructions</button>
+            <div class="modal fade" id="instructionsModal" tabindex="-1" role="dialog" aria-labelledby="purchaseLabel" aria-hidden="true">
```

Comment:
`purchaseLabel`? 🤔 i think you need to change that to something that more accurately describes your elements

Reply:
We've changed this to instructionsLabel